roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -14,33 +14,32 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-
-# "edpm_bootstrap" will search for and load any operating system variable file
-
 - name: Ensure /var/log/journal exists
-  become: true
   ansible.builtin.file:
     path: /var/log/journal
     state: directory
-    mode: 0750
+    mode: '0750'
     owner: root
     group: root
     setype: var_log_t
+  become: true
 
 - name: Check cloud-init status
   ansible.builtin.shell: systemctl is-active cloud-init.service || systemctl is-enabled cloud-init.service
   failed_when: false
-  become: true
   register: cloud_init_enabled
+  become: true
 
-- debug:
+- name: Print cloud-init service status
+  ansible.builtin.debug:
     var: cloud_init_enabled
 
 - name: Check if cloud-init is disabled via kernel args
   ansible.builtin.shell: cat /proc/cmdline | grep -q cloud-init=disabled
```

Comment:
I don't believe this task needs root. The `/proc/cmdline` is 444 by default.
You may also replace the shell with https://docs.ansible.com/ansible/latest/collections/ansible/builtin/lineinfile_module.html in check mode.